python/UniverseWrapper.cpp

```diff
@@ -146,6 +146,16 @@ namespace {
     const Ship::PartMeterMap&
                             (Ship::*ShipPartMeters)(void) const =                               &Ship::PartMeters;
 
+    const ShipDesign*       GetShipDesignByName(const Universe& universe, const std::string& name) {
+        Universe::ship_design_iterator design_it = universe.beginShipDesigns();
+        while (design_it != universe.endShipDesigns()) {
+            if (design_it->second->Name(false) == name)
+                return design_it->second;
+            ++design_it;
+        }
+        return 0;
+    }
```

Comment:
Is there a specific reason why you don't just expose `Universe::GetGenericShipDesign`, which (at least to my eyes) does exactly the same as your helper function here?